.github/ISSUE_TEMPLATE/new-feature-template.md

```diff
@@ -4,7 +4,15 @@ about: Request a new feature
 title: 'Feature: <Briefly describe the feature>'
 labels: new-feature
 assignees: ''
-
 ---
 
-<Describe the feature request>
+_Describe the feature request._
+
+<!--
```

Comment:
Do you think its worth mentioning the link to contribution guide here and encourage users to contribute if they want to?